app/config.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
   // Organisations API
   organisationApiUrl: process.env.ORGANISATION_API_URL || 'http://localhost:5103',
 
+  // Buying Catalogue API
+  buyingCatalogueApiHost: process.env.API_HOST || 'http://localhost:5100',
```

Comment:
we can both get demo on adding these today as I have to add orgapi too!

Reply:
that would be good. We've got a task to add them to the cluster. See if Adrian has any time to go through that today.